### ISSUE TITLE
shot command generates unhandled rejection of status code 431

### ISSUE THREAD

**Issue by Shigma** [koishijs/koishi#64]
**Describe the bug**

**To Reproduce**

```
shot https://browsehappy.com/
```

**Expected behavior**

正常获取网页截图。

**Screenshots**

```
Error: Request failed with status code 431
    at createError (E:\code\touhou\koishi\node_modules\axios\lib\core\createError.js:16:15)
    at settle (E:\code\touhou\koishi\node_modules\axios\lib\core\settle.js:17:12)
    at IncomingMessage.handleStreamEnd (E:\code\touhou\koishi\node_modules\axios\lib\adapters\http.js:236:11)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1244:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

**Versions**

- OS: Windows 10
- Node Version: 14.7.0
- koishi-plugin-puppeteer: 1.0.0-beta.3

**Additional context**

**Comment by Shigma** [comment]
迷惑的是这是个 axios 报错，但无论是 puppeteer-core 还是 koishi-plugin-puppeteer 都没有这个依赖。

**Comment by Shigma** [comment]
Fixed in 33cd79991fc49488c410a7f153aac3000fb93da9.

A lot thanks to @masnn and @shugen002.
